plugins/CoreHome/Tracker/VisitRequestProcessor.php

```diff
@@ -263,4 +268,25 @@ private function getLastKnownActionTime(VisitProperties $visitProperties, $lastK
 
         return $visitProperties->getProperty('visit_last_action_time');
     }
+
+    /**
+     * Returns true if the last user_id did not change.
+     * @return bool
+     */
+    protected function lastUserIdWasSetAndDoesMatch(VisitProperties $visitProperties, Request $request)
```

Comment:
@diosmosis @mattab @MichaelHeerklotz  I'm now actually wondering...  on a regular website when tracking userId, and then the userId changes, would we create a new visit keeping the same visitorId maybe? Just a random thought maybe.

Reply:
Actually, I'm seeing this is what it actually implements. Be just good to have this behaviour confirmed.

Reply:
All good, I totally see we want this behaviour. Was not reading the code right initially.